docs/admin.rst

```diff
@@ -191,14 +191,14 @@ possession when they attempt to log in to SecureDrop.
 
 
 Passphrases and Two-Factor Resets
----------------------------------
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 .. warning:: Both of these operations will lock a user out of their
    SecureDrop accounts. We recommend having users be physically present when
    resetting their passphrase or two-factor authentication. If this is not
-   possible, store passphrases and/or two-factor authentication secret in 
-   your own password manager before securely transmitting them to the user 
-   in question, and delete them once the user has confirmed they can 
+   possible, store passphrases and/or two-factor authentication secret in
```

Comment:
I know this is just a whitespace change but it made me notice it should be "and/or two-factor authentication secret**s**" -- plural.

Reply:
good flag, tbh I think they should all be singular (common case is a single account), done in aa2d96968dc8da9d4556f9836273f2269ad95180